### ISSUE TITLE
Sandboxie-Plus 1.14.8

### ISSUE THREAD

**Issue by gomactrading** [sandboxie-plus/Sandboxie#4240]
### Describe what you noticed and did

Recover Files does not work, at all.  Reverted to 1.14.7 and all is well.

### How often did you encounter it so far?

All the time

### Expected behavior

Recover Files should work

### Affected program

Vivaldi	6.9.3447.44 (Stable channel) (64-bit) 

### Download link

https://github.com/sandboxie-plus/Sandboxie/releases

### Where is the program located?

The program is installed both inside and outside the sandbox.

### Did the program or any related process close unexpectedly?

No, not at all.

### What version of Sandboxie are you running now?

1.14.7

### Is it a new installation of Sandboxie?

I recently did a new clean installation.

### Is it a regression from previous versions?

Only applicable to 1.14.8

### In which sandbox type you have this problem?

In a standard isolation sandbox (yellow sandbox icon).

### Can you reproduce this problem on a new empty sandbox?

I can confirm it also on a new empty sandbox.

### What is your Windows edition and version?

Win 11 64bit

### In which Windows account you have this problem?

A local account (Administrator).

### Please mention any installed security software

Microsoft Defender

### Did you previously enable some security policy settings outside Sandboxie?

No

**Comment by offhub** [comment]
Is the File Recovery window not opening, or does it fail to recover files when you select ‘Recover’ from the File Recovery window? Also, is it giving any errors? If so, please share along with your sandboxie configuration.

**Comment by coltsfootrock** [comment]
I use Vivaldi (same build) on Win 11 64 bit with Defender as AV.
I use quick recovery and could download a file and it offered it to me when I closed the browser.
Immediate recovery did not work for me. However I kept ticking and unticking the box trying downloads and after a few times everything worked okay.

**Comment by gomactrading** [comment]
Recovery window does not show up. Recover files does nothing.  To recover files I go to Box Content -> Explore Content -> user -> current and move files from there.  I reverted to 1.14.6 and I do not have to go through any pain to recover files.  I hope this will get fixed soon.

On Friday, September 13, 2024 9:57:22 AM (-04:00), coltsfootrock wrote:



I use Vivaldi (same build) on Win 11 64 bit with Defender as AV.
I use quick recovery and could download a file and it offered it to me when I closed the browser.
Immediate recovery did not work for me. However I kept ticking and unticking the box trying downloads and after a few times everything worked okay.

**Comment by gomactrading** [comment]
File Recovery window does not open and, therefore, there are no errors given. Recover files does absolutely nothing.  I reverted back to 1.14.6 with no problems whatsoever using same sandboxie and sandboxie-plus configurations (working in portable mode)

On Friday, September 13, 2024 4:46:35 AM (-04:00), offhub wrote:



Is the File Recovery window not opening, or does it fail to recover files when you select ‘Recover’ from the File Recovery window? Also, is it giving any errors? If so, please share along with your sandboxie configuration.

**Comment by offhub** [comment]
Can you try with version 1.14.7 to see if the same issue occurs?

https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.14.7

**Comment by gomactrading** [comment]
No problem in 1.14.7.  Works as intended.  Thank you.

On Friday, September 13, 2024 1:57:44 PM (-04:00), offhub wrote:



Can you try with version 1.14.7 to see if the same issue occurs?

https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.14.7

**Comment by DavidXanatos** [comment]
This is very strange the only changes between 1.14.7 and 1.14.8 is a fix for the cert and a change to the box export option, it cant affect recovery.
Can anyone else reproduce this issue?

**Comment by gomactrading** [comment]
You are correct.  After reinstalling 1.14.6 followed by 1.14.7 and using them as intended, I reinstalled 1.14.8, after receiving your email, and no longer have any problems. The only explanation I have is that, this time,  I ran the unpacking as administrator. I do not recall if I did the same, before reporting the problem a few days ago, but I usually install everything (including portable) as administrator. Anyway, for now, the issue #4240 does not seem to be an issue, anymore.  Will let you know  if I run into this, again, while using the sandbox more extensively.  Thank you.

On Sunday, September 15, 2024 7:06:40 AM (-04:00), DavidXanatos wrote:



This is very strange the only changes between 1.14.7 and 1.14.8 is a fix for the cert and a change to the box export option, it cant affect recovery.
Can anyone else reproduce this issue?
